io.github.openWMail.appdata.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.github.openwmail.desktop</id>
+  <name>openWMail</name>
+  <project_license>MPL 2.0</project_license>
+  <developer_name>openWMail</developer_name>
+  <summary>Open Source Email Client</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <url type="homepage">https://openwmail.github.io</url>
+  <launchable type="desktop-id">io.github.openwmail.desktop</launchable>
+  <update_contact>https://github.com/openWMail/openWMail/issues</update_contact>
+  <description>
+    <p>openWMail the free and open-source desktop client for Gmail and Google Inbox</p>
```

Comment:
Sounds more like a summary; "Open Source Email Client" sounds like general purpose client while it isn't.